### ISSUE TITLE
Test images link doesn't work

### ISSUE THREAD

**Issue by rnk-src** [advimman/lama#261]
I'm not sure if it's just me, but it says that the images link is also closed down for now because of the user's disk space, just like with the models link. Although a recent issue was closed with the correct link for the models, is there a correct link for the test images?

edit: issue #260 has the new model link

**Comment by zhuang2002** [comment]
Have you solved the problem

**Comment by rnk-src** [comment]
Hey! Sorry for the late response, I really didn't actually need the test images in the end. I just took an image and ran it through the generate_mask script. 

"python3 bin/gen_mask_dataset.py configs/data_gen/random_thick_512.yaml LaMa_test_images/ LaMa_test_images/ --n-jobs=5 --ext=png" is what i used after creating a fork, making a folder called "LaMa_test_images", and putting in the image that i made a PNG. After that, I was able to test my program. Sorry I couldn't provide a solution to the test images though :(.
